state/application.go

```diff
@@ -3490,24 +3490,6 @@ func (a *Application) UnitNames() ([]string, error) {
 	return u, errors.Trace(err)
 }
 
-// CharmPendingToBeDownloaded returns true if the charm referenced by this
-// application is pending to be downloaded.
-func (a *Application) CharmPendingToBeDownloaded() bool {
```

Comment:
Funny that these just didn't have any tests.

Reply:
This is because we recently removed all tests for `state`